src/content/en/fundamentals/performance/http2/index.md

```diff
@@ -254,7 +254,7 @@ reassemble them on the other end.
 ![HTTP/2 request and response multiplexing within a shared connection](images/multiplexing01.svg)
 
 The snapshot captures multiple streams in flight within the same connection. The
-client is transmitting a DATA frame (stream 5) to the server, while the server
+client is transmitting a `DATA` frame (stream 5) to the server, while the server
```

Comment:
Unless I'm misreading the HTTP2 spec, DATA as it's used here is nothing that will every appear in code. Meaning it's more like 'response header' than '`Response` instance'. (Notice the formatting.) Please remove the backticks.